tutorial/practicals/nemo/3_nemo_openmp/README.md

```diff
@@ -3,12 +3,11 @@
 This tutorial follows on from Tutorials 1 and 2
 (../1_nemo_psyir/README.md and ../2_nemo_profiling/README.md) and
```

Comment:
Could be converted to "Tutorial 1" and "Tutorial 2" links

Reply:
Oh yes. I tried to think of a way of phrasing it to do that but failed - I like your suggestion.